O3exp/pipeline/analysis.ini

```diff
@@ -237,27 +246,27 @@ f-lower = ${inspiral|low-frequency-cutoff}
 log-param = True False False
 smoothing-width = 0.4 0.2 0.08
 
-[distribute_background_bins]
+;[distribute_background_bins]
 
 [multiifo_coinc]
 ; additional time (in seconds) to add to light-travel time to construct time coincidence window
-; only 2- or multi-ifo coinc times are considered to construct the time shift background ("strict coincidence")
 coinc-threshold = 0.002
-ranking-statistic = exp_fit_sg_csnr
+ranking-statistic = newsnr_sgveto
+; this ranking statistic is _not_ optimised for best results, but will work
 randomize-template-order =
 
 [multiifo_coinc-full_data]
 ; time-slide interval in seconds
 timeslide-interval = 0.1
 ; successive levels of decimation to reduce memory load for background coincs below stated values
-loudest-keep-values = 9:10 8.5:30 8:30 7.5:30
+loudest-keep-values = 15:10 10:30 5:30 0:30
```

Comment:
Are these values appropriate for the newsnr_sgveto statistic?

Reply:
It looks like the 15,10,5,0 values are for the new fg/bg statistic and if one is using a 'newsnr-like' statistic for workflow testing the decimation should be left as it was

Reply:
Good point, I'll sort that now